components/PetNav.js

```diff
@@ -18,19 +34,20 @@ const StyledNav = styled.nav`
   justify-content: space-evenly;
 `;
 
-const StyledButtonInactive = styled.button`
+const StyledLink = styled(Link)`
+  text-decoration: none;
+`;
+
+const StyledButton = styled.button`
   margin: 15px;
   font-family: inherit;
   font-size: 1rem;
   background-color: white;
   padding: 5px 20px;
   width: 160px;
   border-radius: 3px;
-  border: 3px solid #aaa;
-`;
-
-const StyledButtonActive = styled(StyledButtonInactive)`
-  border: none;
-  background-color: #5885da;
-  color: white;
-`;
+  border: ${(props) => (props.$active ? "none" : "3px solid #aaa")};
+  background-color: ${(props) => (props.$active ? "#5885aa" : "white")};
+  color: ${(props) => (props.$active ? "white" : "black")};
```

Comment:
combine all three properties into one callback for improved readability. Set default values first, then overwrite them if the prop $active is true:
```suggestion
  border: 3px solid #aaa;
  background-color: white;
  color: black;
  
  ${(props) => props.$active && css`
    border: none;
    background-color: #5885aa;
    color: white;
  
  `}
```